src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -743,6 +758,49 @@ private void handleViewPinnedMessages() {
     startActivity(intent);
   }
 
+  private void handleSearch(MenuItem item) {
+    bottomPanel = (InputPanel) findViewById(R.id.bottom_panel);
```

Comment:
I would remove all spaces after casting here, that is the style that is in the rest of the repo and even this pr.